src/games/strategy/triplea/ui/MapPanel.java

```diff
@@ -128,25 +126,25 @@ public void run() {
         });
       }
     });
-    recreateTiles(data, m_uiContext);
-    m_uiContext.addActive(new Active() {
+    recreateTiles(data, this.uiContext);
```

Comment:
Well inthis case it is neccessary... this variable should refer the uiContext of the class, not of the method